point/gemini_commands.py

```diff
@@ -609,29 +549,29 @@ def interpret(self):
         # (e.g. angle ranges:  [0,180) or [-90,+90] or [0,360)  etc)
         fields = self.get_raw()
         self._values = dict()
-#        self._values['phys_x']       = parse_int           (fields[ 0])  # raises IntegerParseError on failure
-#        self._values['phys_y']       = parse_int           (fields[ 1])  # raises IntegerParseError on failure
-        self._values['pra']          = parse_int           (fields[ 0])  # raises IntegerParseError on failure
-        self._values['pdec']         = parse_int           (fields[ 1])  # raises IntegerParseError on failure
-        self._values['ra']           = parse_ang_dbl       (fields[ 2])  # raises DblPrecAngleParseError on failure
-        self._values['dec']          = parse_ang_dbl       (fields[ 3])  # raises DblPrecAngleParseError on failure
-        self._values['ha']           = parse_ang_dbl       (fields[ 4])  # raises DblPrecAngleParseError on failure
-        self._values['az']           = parse_ang_dbl       (fields[ 5])  # raises DblPrecAngleParseError on failure
-        self._values['alt']          = parse_ang_dbl       (fields[ 6])  # raises DblPrecAngleParseError on failure
+#        self._values['phys_x']       = parse_int           (fields[ 0])  # raises G2ResponseIntegerParseError on failure
```

Comment:
I'll accept this for now, but I'd really like to move towards proper docstrings on the functions themselves, with a `Raises:` section to indicate what exceptions it can raise.